srcpkgs/nautilus/template

```diff
@@ -21,7 +21,7 @@ homepage="https://wiki.gnome.org/Nautilus"
 #changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/master/NEWS"
 changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-44/NEWS"
```

Comment:
This should be: `https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-45/NEWS`

Reply:
Done, thanks.